examples/prompts.llmake

```diff
@@ -0,0 +1,20 @@
+_global:
+    text: "This is a shared context that will apply to multiple prompts."
+
+landscape: _global
+    text: "Describe a natural landscape with mountains and rivers."
```

Comment:
this file still uses the old syntax, if if it is no longer needed you can either remove it, or turn it into a test